cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -389,1235 +379,126 @@ data:
     {{ end }}
     {{- if .Values.oidc.enabled }}
         location /auth {
-            proxy_pass http://aggregator/isAuthenticated;
+            proxy_pass http://model/isAuthenticated;
         }
         # This location is used to remain compatible with older configurations of Kubecost SSO. All SSO logic should now be handled by the Aggregator container.
         location /model/auth {
-            proxy_pass http://aggregator/isAuthenticated;
+            proxy_pass http://model/isAuthenticated;
         }
     {{- end }}
     {{- if .Values.saml.enabled }}
         location /auth {
-            proxy_pass http://aggregator/isAuthenticated;
+            proxy_pass http://model/isAuthenticated;
         }
         # This location is used to remain compatible with older configurations of Kubecost SSO. All SSO logic should now be handled by the Aggregator container.
         location /model/auth {
-            proxy_pass http://aggregator/isAuthenticated;
+            proxy_pass http://model/isAuthenticated;
         }
     {{- if .Values.saml.rbac.enabled }}
         location /authrbac {
-            proxy_pass http://aggregator/isAdminAuthenticated;
+            proxy_pass http://model/isAdminAuthenticated;
         }
         # This location is used to remain compatible with older configurations of Kubecost SSO. All SSO logic should now be handled by the Aggregator container.
         location /model/authrbac {
-            proxy_pass http://aggregator/isAdminAuthenticated;
+            proxy_pass http://model/isAdminAuthenticated;
         }
     {{- end }}
     {{- end }}
 
 
 {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
-    # TODO make aggregator route the default, start special-casing
```

Comment:
Thanks for deleting all this. Looks like all the lines below simply route specific `/model/` endpoints to the Aggregator pod.

Reply:
the remaining endpoints here actually route to the cloud cost pod, which is why they need to stay